google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/descriptor/AppEngineWebSchemaProvider.java

```diff
@@ -73,10 +74,14 @@ public XmlFile getSchema(@NotNull @NonNls String url, @Nullable Module module,
       if (facet != null) {
         final File file;
         if (isApplicationXmlFile(baseFile)) {
-          file = facet.getSdk().getApplicationSchemeFile();
+          file = CloudSdkService.getInstance().getApplicationSchemeFile();
         } else {
-          file = facet.getSdk().getWebSchemeFile();
+          file = CloudSdkService.getInstance().getWebSchemeFile();
```

Comment:
ah I think I remember what this is. It's for multi-module/service java apps that are bundled as EARs.  Only appcfg supports it so it probably can be safely removed. People might be a little surprised when they find that deployment doesn't chain all their separate services..

@ludoch